src/cmd/gz.cc

```diff
@@ -22,7 +22,7 @@
 #include <string.h>
 #include <vector>
 
-#include "sdf/config.hh"
+#include "sdf/sdf_config.h"
```

Comment:
I think we actually want `sdf/config.hh`.  https://github.com/gazebosim/sdformat/blob/33b8c91cbbd0ad6f7f8e35d6693ab120192c17c7/include/sdf/sdf_config.h#L21-L23